common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -341,92 +240,74 @@ nav.sequence-nav {
   }
 }
 
-.seq_contents {
-  display: none;
-}
-
-nav.sequence-bottom {
-  margin: lh(2) 0 0;
-  text-align: center;
-
-  ul {
-    @include clearfix();
-    display: inline-block;
-
-    li {
-      @include float(left);
-      width: 50px;
-      height: 44px;
-      border: 1px solid $gray-l3;
-      @include linear-gradient(top, #eee, #ddd);
-      box-shadow: 0 1px 0 rgba(255, 255, 255, .7) inset;
-
-      &.prev, &.next {
-        margin-bottom: 0;
-
-        a {
-          background-position: center center;
-          background-repeat: no-repeat;
-          border: none;
-          display: block;
-          padding: lh(0.5) 4px;
-          text-indent: -9999px;
-          overflow: hidden;
-          @include transition(all .2s $ease-in-out-quad 0s);
-
-          &:hover, &:focus {
-            opacity: 0.5;
-            background-position: center 15px;
-          }
-
-          &.disabled {
-            opacity: 0.4;
-          }
-
-/*          &:focus {
-            outline: 0;
-          }
-*/        }
-      }
+.sequence-nav-button {
+  @include linear-gradient(top, #eee, #ddd);
+  @include transition(all .2s $ease-in-out-quad 0s);
+  position: absolute;
+  top: 0;
+  z-index: 50;
```

Comment:
If you're going to introduce a ``z-index`` rule here, let's use LMS's Sass scale (https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Fbase%2F_mixins.scss#L93)?